### PR TITLE
Fixes after review:

### PR DESCRIPTION
**Pull request by gotlieb**
protocols and flavorIds to be comma separated string
remove priority parameter
remove storageId from manifest url and add deliveryProfileId to url
renaming of classes
fixes following code review
PLAT-6283

### CODE REVIEW

File: alpha/apps/kaltura/lib/kPlaybackContextDataHelper.php [hunk 1]
```diff
@@ -298,23 +293,12 @@ private function constructLocalPlaybackSources(entry $dbEntry, kContextDataHelpe
 
 			if (count($playbackFlavors))
 			{
-				$manifestUrl = myEntryUtils::buildManifestUrl($dbEntry, $deliveryProfile->getStreamerType(), $playbackFlavors, null);
-				$this->localPlaybackSources[] = new kPlaybackSource($deliveryProfile->getId(), $deliveryProfile->getStreamerType(), $this->getLocalPriority($deliveryProfile), $this->constructProtocol($deliveryProfile), array_keys($playbackFlavors), $manifestUrl, $drmData);
+				$manifestUrl = myEntryUtils::buildManifestUrl($dbEntry, $deliveryProfile->getStreamerType(), $playbackFlavors, $deliveryProfile->getId());
+				$this->localPlaybackSources[] = new kPlaybackSource($deliveryProfile->getId(), $deliveryProfile->getStreamerType(), $this->constructProtocol($deliveryProfile), implode("','", array_keys($playbackFlavors)), $manifestUrl, $drmData);
```
Comment (MosheMaorKaltura) [inline alpha/apps/kaltura/lib/kPlaybackContextDataHelper.php]: Fix ','

File: alpha/apps/kaltura/lib/kPlaybackContextDataHelper.php [hunk 2]
```diff
@@ -349,24 +333,13 @@ private function constructRemotePlaybackSources(entry $dbEntry, kContextDataHelp
 					foreach ($filteredDeliveryProfileFlavorsForDc as $flavorAssetForDc)
 						$dcFlavorIds[] = $flavorAssetForDc->getId();
 
-					$manifestUrl = myEntryUtils::buildManifestUrl($dbEntry, $deliveryProfile->getStreamerType(), $filteredDeliveryProfileFlavorsForDc, $dcId);
-					$this->remotePlaybackSources[] = new kPlaybackSource($deliveryProfile->getId(), $deliveryProfile->getStreamerType(), $this->getRemotePriority($deliveryProfile), $this->constructProtocol($deliveryProfile), $dcFlavorIds, $manifestUrl, $flavorToDrmData);
+					$manifestUrl = myEntryUtils::buildManifestUrl($dbEntry, $deliveryProfile->getStreamerType(), $filteredDeliveryProfileFlavorsForDc, $deliveryProfile->getId());
+					$this->remotePlaybackSources[] = new kPlaybackSource($deliveryProfile->getId(), $deliveryProfile->getStreamerType(), $this->constructProtocol($deliveryProfile), implode("','", array_keys($dcFlavorIds)), $manifestUrl, $flavorToDrmData);
```
Comment (MosheMaorKaltura) [inline alpha/apps/kaltura/lib/kPlaybackContextDataHelper.php]: ','

File: alpha/apps/kaltura/lib/myEntryUtils.class.php [hunk 3]
```diff
@@ -1578,7 +1575,7 @@ public static function getFileExtensionByFormat($format, $fileExtension = "" )
 			case PlaybackProtocol::AKAMAI_HDS:
 				return PlaybackProtocol::AKAMAI_HDS . "/manifest.f4m";
 			case PlaybackProtocol::HTTP:
-				return PlaybackProtocol::HTTP . "/name/a.$fileExtension";
+				return "url/name/a.$fileExtension";
```
Comment (MosheMaorKaltura) [inline alpha/apps/kaltura/lib/myEntryUtils.class.php]: Use enum instead of const

File: api_v3/lib/types/KalturaPlaybackContext.php [hunk 4]
```diff
@@ -16,28 +16,22 @@ class KalturaPlaybackContextResult extends KalturaObject{
...
-	/**
 	 * Array of actions as received from the rules that invalidated
 	 * @var KalturaRuleActionArray
 	 */
 	public $actions;
 
 	/**
 	 * Array of actions as received from the rules that invalidated
-	 * @var KalturaPlaybackRestrictionsArray
+	 * @var KalturaAccessControlMessageArray
```
Comment (tan-tan-kanarek) [inline api_v3/lib/types/KalturaPlaybackContext.php]: I don't understand why the code should be accompanied with the message, the actions array should be enough.

**Pull request merged**